packer/linux/scripts/install-lifecycled.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu -o pipefail
+
+LIFECYCLED_VERSION=v3.2.0
+
+MACHINE=$(uname -m)
+
+case "${MACHINE}" in
+	x86_64)    ARCH=amd64;;
+	aarch64)   ARCH=arm64;;
+	*)         ARCH=unknown;;
+esac
+
+echo "Installing lifecycled ${LIFECYCLED_VERSION}..."
+
+sudo touch /etc/lifecycled
+sudo curl -Lf -o /usr/bin/lifecycled \
+	https://github.com/lox/lifecycled/releases/download/${LIFECYCLED_VERSION}/lifecycled-linux-${ARCH}
```

Comment:
although this still works, I think it should point to https://github.com/buildkite/lifecycled (it redirect to this one)

Reply:
Updated in https://github.com/buildkite/elastic-ci-stack-for-aws/pull/922/commits/0388a19706fe7f7c4de4d3ee59a86979f2782e7c